packages/blade/src/components/Accordion/Accordion.stories.internal.tsx

```diff
@@ -0,0 +1,109 @@
+const meta: Meta<AccordionProps> = {
+  title: 'Components/Accordion (Internal)',
+  component: AccordionComponent,
+  args: {},
+  argTypes: {
+    ...getStyledPropsArgTypes(),
+  },
+  parameters: {
+    docs: {
+      page: Page,
+    },
+  },
+};
+
+const AccordionTemplate: ComponentStory<typeof AccordionComponent> = ({ ...args }) => {
+  return (
+    <AccordionComponent {...args}>
+      <AccordionItem
+        title="Appetite for Destruction Appetite for Destruction Appetite for Destruction Appetite for Destruction"
+        description={`Appetite for Destruction is the debut studio album by American hard rock band Guns N' Roses, released by Geffen Records on July 21, 1987. It initially received little mainstream attention, and it was not until the following year that Appetite for Destruction became a commercial success, after the band had toured and received significant airplay with the singles "Welcome to the Jungle", "Paradise City", and "Sweet Child o' Mine".`}
+      />
+      <AccordionItem
+        title="Appetite for Destruction Appetite for Destruction Appetite for Destruction Appetite for Destruction"
+        description={`Appetite for Destruction is the debut studio album by American hard rock band Guns N' Roses, released by Geffen Records on July 21, 1987. It initially received little mainstream attention, and it was not until the following year that Appetite for Destruction became a commercial success, after the band had toured and received significant airplay with the singles "Welcome to the Jungle", "Paradise City", and "Sweet Child o' Mine".`}
+      />
+      <AccordionItem
+        title="Appetite for Destruction Appetite for Destruction Appetite for Destruction Appetite for Destruction"
+        description={`Appetite for Destruction is the debut studio album by American hard rock band Guns N' Roses, released by Geffen Records on July 21, 1987. It initially received little mainstream attention, and it was not until the following year that Appetite for Destruction became a commercial success, after the band had toured and received significant airplay with the singles "Welcome to the Jungle", "Paradise City", and "Sweet Child o' Mine".`}
+      />
+    </AccordionComponent>
+  );
+};
+
+const AccordionWithIconsTemplate: ComponentStory<typeof AccordionComponent> = ({ ...args }) => {
+  return (
+    <AccordionComponent {...args}>
+      <AccordionItem
+        title="Appetite for Destruction Appetite for Destruction Appetite for Destruction Appetite for Destruction"
+        description={`Appetite for Destruction is the debut studio album by American hard rock band Guns N' Roses, released by Geffen Records on July 21, 1987. It initially received little mainstream attention, and it was not until the following year that Appetite for Destruction became a commercial success, after the band had toured and received significant airplay with the singles "Welcome to the Jungle", "Paradise City", and "Sweet Child o' Mine".`}
```

Comment:
Let's use normal strings instead of template literals.

Reply:
Should be fine I think, we should let prettier / eslint take care of these 😅 

I've also updated all stories in the latest PR #1296